proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "celestia-proto"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Wouldn't this be increased by release-plz?

Reply:
commited by mistake when I was testing release-plz, removing.